docs/user/dashboard/lens.asciidoc

```diff
@@ -211,8 +213,12 @@ image::images/lens_annotations_8.2.0.png[Lens annotations]
 
 . Change the *Appearance* options for how you want the annotation to display.
 
+. To add a field to the tooltip, click *Add field*, then select the field you want to display.
```

Comment:
The tooltip configuration appears on the query based annotations that we don't mention here. I propose to also mention them and then mention the tooltip fields. Otherwise it will be confusing for the users